config/routes.rb

```diff
@@ -3,4 +3,11 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root 'users#index'
+
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show'
+
+  get '/users/:user_id/posts', to: 'posts#index'
+  get '/users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] **[OPTIONAL]** Instead of specifying one by one each route of your application which can get messy really quick, I suggest you use one of the Rails helpers: `resources` which creates RESTful routes for your controllers!

```suggestion
  root 'users#index'
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end



```

In this case, we specify only `:index` and `:show` actions to not create routes that we're not using yet! 🤓 🔧